addons/mail/static/src/discuss/core/common/channel_member_list.xml

```diff
@@ -34,6 +34,7 @@
                 <ImStatus member="member" className="'position-absolute top-100 start-100 translate-middle mt-n1 ms-n1'"/>
             </div>
             <span t-ref="displayName" class="ms-2 text-truncate" t-esc="member.name"/>
+            <span t-if="member.is_owner" class="p-1 ms-1" title="Channel Owner" aria-label="Channel Owner">👑</span>
```

Comment:
I would `fs-2`, the apple version of emoji is quite dense.
No `fs-2` (current code):
<img width="185" alt="Screenshot 2024-09-05 at 17 38 19" src="https://github.com/user-attachments/assets/3b7acca9-8da9-4a80-818a-8ebf9a4da16f">
With `fs-2`:
<img width="190" alt="Screenshot 2024-09-05 at 17 38 06" src="https://github.com/user-attachments/assets/3eb0f1fc-e7a5-4bb6-921e-4a6275d1cdf5">

Reply:
Someone can troll and put the crown emoji suffix in name:
<img width="398" alt="Screenshot 2024-09-05 at 17 48 53" src="https://github.com/user-attachments/assets/ac877879-3778-41fc-b423-b11e025ce74a">
_(Mitchell Admin is owner, but Marc Demo is not)_

Mouse hover title helps a bit so there's that, but would be nice to have a style that is more unique. The `fs-2` makes it a bit more distinct:
<img width="202" alt="Screenshot 2024-09-05 at 18 04 56" src="https://github.com/user-attachments/assets/0260c8c7-1e90-47c8-bfa7-8aadeafbfed7">


_To bad we're still on font-awesome version 4. if we had a more up-to-date font-awesome, we could have used `fa-crown`_

Reply:
I feel like the fs-2 is too heavy in linux and windows. Even if in mac, it looks too huge already... 
![5825871446](https://github.com/user-attachments/assets/0948371b-20cf-4264-b832-f12396097bfb)

Reply:
actually, we don't really care about trolling since all the related rights are not based on the icon... and we can even have a catalog in the future to represent these owners. But indeed, the `fa-crown` is the best solution but we don't have that

Reply:
Honestly I would not go with emoji, it doesn't look great.
I looked at font-awesome 4 icons that could look nice. `fa-shield`, `fa-star`, `fa-lock` are the 3 best icons I could find. I don't have strong preference for any of them: `fa-shield` is the most uniquely appropriate, `fa-star` gives a generic visual of "important", and `fa-lock`  gives some admin vibes.

I prefer them better than the emoji crown solution.

<img width="208" alt="Screenshot 2024-09-09 at 20 39 13" src="https://github.com/user-attachments/assets/0c934514-402e-4515-9ce8-79efebbc6a7e">
<img width="191" alt="Screenshot 2024-09-09 at 20 39 24" src="https://github.com/user-attachments/assets/811904aa-f668-4d62-be60-fc1ac4717535">
<img width="201" alt="Screenshot 2024-09-09 at 20 39 52" src="https://github.com/user-attachments/assets/daefd4e7-0ffe-431d-b2a2-a6b2976b582d">
<img width="193" alt="Screenshot 2024-09-09 at 20 40 45" src="https://github.com/user-attachments/assets/33acf450-b697-4f55-8221-5fe52ecca18f">
<img width="199" alt="Screenshot 2024-09-09 at 20 41 01" src="https://github.com/user-attachments/assets/2ff1c339-9c32-4f51-93a4-47d68471b57b">
<img width="198" alt="Screenshot 2024-09-09 at 20 41 15" src="https://github.com/user-attachments/assets/9f8f2b60-707d-422e-9cc8-7466747f78a1">

Reply:
`fa-star` can be a good solution indeed

Reply:
but it can conflict with future features ( might be? I don't know, just star means too generic to me but it is kind of good here indeed )

Reply:
There's plan to improve our icons in a few months: https://www.odoo.com/odoo/discuss/project.task/3714335

So better find a good fit now, the near future is bright as we would be able to change to a better fit ;)

Reply:
> and also, I wish they fix the trump ear problem with the task as well (hopefully)

I think the solution would be to combine/stack `fa|bi-slash` and `fa|bi-headphone`. We lack the slash in Font Awesome 4, but newer icons from either solution will let us put an alternative to trump ear :D